code/__HELPERS/logging/debug.dm

```diff
@@ -38,30 +36,26 @@
 
 /// Logging for hard deletes
 /proc/log_qdel(text)
-	WRITE_LOG(GLOB.world_qdel_log, "QDEL: [text]")
-
-/// Logging for SQL errors
-/proc/log_query_debug(text)
-	WRITE_LOG(GLOB.query_debug_log, "SQL: [text]")
+	logger.Log(LOG_CATEGORY_DEBUG_QDEL, text)
 
 /* Log to the logfile only. */
 /proc/log_runtime(text)
-	WRITE_LOG(GLOB.world_runtime_log, text)
+	logger.Log(LOG_CATEGORY_DEBUG_RUNTIME, text)
```

Comment:
lets leave runtime logs off, or maybe just do both for this one: runtime.log and runtime.json. Its gonna take me way too long to add support for parsing new runtimes given i have to edit both php and c++ code to do so and the c++ code is a single c file with no deps so i'd rather not have to figure out how to add json support.

we have dd.log as well, but thats unsanitized and includes things that can't be trusted to coders/players in public logs and also doesn't have the same timestamp format `rc.cpp` expects